src/stacked_hourglass/predictor.py

```diff
@@ -14,8 +14,12 @@ def _check_batched(images):
     return False
 
 
+# Output is <this> x smaller than input
+MODEL_DOWNSCALE_FACTOR = 4
```

Comment:
Do you think that this is actually necessary? I can see that you were trying to preserve the existing way of doing things, but could `output_shape` be inferred from the shape of `heatmaps` instead?

Reply:
The main reason being that in theory `HumanPosePredictor` should be agnostic of specific details like the output stride of `model`.

Reply:
I agree it would be better without needing to hardcode the output heatmap size. This is a really simple change!